src/utils.ts

```diff
@@ -25,6 +25,16 @@ export default {
       })
     }
   },
+  extractBookmarkId: (to, type) => {
+    if (to.hash) {
+      const regex = new RegExp(`#${type}:\\d+`, 'g' )
+      let match = to.hash.match(regex)
+      match = _.size(match) && match[0].match(/\d+/g)
+      return _.size(match) && match[0]
```

Comment:
@johncrossman FWIW, this might be a bit easier to follow with a capture group in the regex. I think if you expressed the regex as #${type}:(\\d+)`, then you could get just the digit portion in `match[1]`.